models/audit_log_entry.go

```diff
@@ -82,13 +83,25 @@ func NewAuditLogEntry(tx *storage.Connection, instanceID uuid.UUID, actor *User,
 }
 
 func FindAuditLogEntries(tx *storage.Connection, instanceID uuid.UUID, filterColumns []string, filterValue string, pageParams *Pagination) ([]*AuditLogEntry, error) {
-	q := tx.Q().Where("instance_id = ?", instanceID)
+	q := tx.Q().Order("created_at desc").Where("instance_id = ?", instanceID)
 
 	if len(filterColumns) > 0 && filterValue != "" {
 		lf := "%" + filterValue + "%"
-		for _, col := range filterColumns {
-			q = q.Where(fmt.Sprintf("payload->>'$.%s' COLLATE utf8mb4_unicode_ci LIKE ?", col), lf)
+
+		builder := bytes.NewBufferString("(")
+		values := make([]string, len(filterColums))
+
+		for idx, col := range filterColumns {
+			builder.WriteString(fmt.Sprintf("payload->>'$.%s' COLLATE utf8mb4_unicode_ci LIKE ?", col))
+			values = append(values, lf)
+
+			if idx+1 < len(filterColumns) {
+				builder.WriteString(" OR ")
```

Comment:
I don't love index math like this. I'd much prefer to use `strings.Join`, but I understand this is slightly better from a performance standpoint.